<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pubsub</class>
 <widget class="QWidget" name="Pubsub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="_horizontalLayout">
     <item>
      <widget class="QLabel" name="_recvLabel">
       <property name="text">
        <string>订阅主题:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_recvLineEdit"/>
     </item>
     <item>
      <spacer name="_horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="_sendLabel">
       <property name="text">
        <string>发送主题:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_sendLineEdit"/>
     </item>
     <item>
      <spacer name="_horizontalSpacer3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_pushButton1">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="_horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_pushButton2">
       <property name="text">
        <string>清理</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="_horizontalSpacer4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="_splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTextBrowser" name="_textBrowser"/>
     <widget class="QPlainTextEdit" name="_plainTextEdit"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
